.circleci/config.yml

```diff
@@ -148,7 +148,6 @@ jobs:
       - image: decidim/decidim@sha256:c0588c1f425fa2433733978fbfa147b4e6cc54be9f6ac4583920158d6a408972
         environment:
           DATABASE_USERNAME: postgres
-          CENSUS_URL: http://mycensus:3001
```

Comment:
Mmm, I think this are necesary to reuse saved VCR cassettes

Reply:
Gravity forms does not use vcr cassettes and does not connect to census as I recall.

Reply:
ah, ok :+1: